dask_geopandas/core.py

```diff
@@ -389,6 +389,60 @@ def morton_distance(self, p=15):
 
         return distances
 
+    def shuffle(self, by="hilbert", column=None, drop=True, npartitions=10, **kwargs):
+
+        """
+        A function that spatially shuffles a Dask-GeoSeries object by a method
+        or a user-defined column
+
+        Parameters
+        ----------
+        by : str
+            partitioning method
+        column : str
+            precomputed column
+        drop : bool
+            to drop the partitioning information held in index
+        npartitions : int
+            number to partition DataFrame
+
```

Comment:
`**kwargs` are missing from the docstring.

Also, I would probably expose `p` directly and pass `**kwargs` to `set_index`.